app/views/operationsAndFunds/WhyNoBankStatementView.scala.html

```diff
@@ -0,0 +1,59 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import controllers.operationsAndFunds.routes._
+@import models.Mode
+@import views.ViewUtils._
+@import config.FrontendAppConfig
+
+
+@this(
+    govukLayout: templates.GovukLayoutWrapper,
+    formHelper: FormWithCSRF,
+    errorSummary: components.errorSummary,
+    button: components.button,
+    h1: components.h1,
+    fieldset: components.fieldset,
+    inputCharacterCount: components.inputCharacterCount,
+)
+
+@(form: Form[_], mode: Mode)(implicit request: Request[_], messages: Messages, appConfig: FrontendAppConfig)
+
+@govukLayout(pageTitle = Some(title(form, "whyNoBankStatement.title", Some("operationsAndFunds.section")))) {
+
+    @formHelper(action = WhyNoBankStatementController.onSubmit(mode)) {
+
+
+        @errorSummary(form.errors)
+        @fieldset("whyNoBankStatement.heading"){
+
+            @inputCharacterCount(
+                form,
+                id = "value",
+                name = "value",
+                rows = 5,
+                value = Some("value"),
+                maxLength = Some(255),
+                isPageHeading = false,
```

Comment:
axe detects no label - make this isPageHeading to true, set label to whyNoBankStatement.heading, and delete the @fieldset tag